### PR TITLE
feat: join growing community section

### LINKED ISSUE #6: Create "Join a growing community" section
See Figma for layout, message Claire (@shivermist) for assets.

### PR DESCRIPTION
**Pull request by owengretzinger**
### TL;DR

Created the join a growing community section. Closes https://github.com/deltahacks/landing-11/issues/6.

### What changed?

- Created a new `JoinGrowingCommunity.tsx` component with the content for the section
- Created a `Header.tsx` component (and changed other sections to use this component)
- Added the section to `index.tsx`

### Other Notes

- I made changes to `index.tsx` so that the purple background is a lower z-index. This way, we can easily add `-z-10` to graphics and they will automatically show up in front of the purple background, but stay behind any other elements such as text.

### Screenshots

Desktop:

Mobile:

Extra Wide Screen:

### How to test?

- Pull the code for this PR and make sure everything looks right.

### CODE REVIEW

**Comment by owengretzinger** [comment]
@Shivermist assets for yellow and red dinos are good but the blue dino and the trails I got from Figma are pretty low res. Do you have higher quality ones you could send?

**Comment by demanr** [comment]
When I look at this on my monitor, the yellow dinos "trail" doesnt extend to the edge of the screen on the right 
image

**Comment by owengretzinger** [comment]
> When I look at this on my monitor, the yellow dinos "trail" doesnt extend to the edge of the screen on the right 

What should we do about this? @Shivermist are you able to extend the trail? Or should I just stretch it out more on wide screens?

**Comment by Krish120003** [comment]
Waiting for #2 before reviewing this

**Comment by Shivermist** [comment]
> @Shivermist assets for yellow and red dinos are good but the blue dino and the trails I got from Figma are pretty low res. Do you have higher quality ones you could send?

Hi 

> @Shivermist assets for yellow and red dinos are good but the blue dino and the trails I got from Figma are pretty low res. Do you have higher quality ones you could send?

Hi Owen! I've uploaded high quality versions of the assets on the google drive under "Design --> Website assets" and the trails are under it's own folder there. Also the trials are supposed to svgs unless there's some kind of visual artifacts that makes it look weird. Do let me know if that's the case. Thanks!

**Comment by Shivermist** [comment]
> > When I look at this on my monitor, the yellow dinos "trail" doesnt extend to the edge of the screen on the right
> 
> What should we do about this? @Shivermist are you able to extend the trail? Or should I just stretch it out more on wide screens?

I've also extended the trails considerably on the Figma (assets also available on the drive). If it's still not long enough, feel free to stretch it out to fit on the screen.

**Comment by owengretzinger** [comment]
Thanks @Shivermist! I updated this PR to use assets from the drive.

@kabirsg I also fixed that issue with the trail showing up over the tree.

**Comment by kabirsg** [comment]
> Thanks @Shivermist! I updated this PR to use assets from the drive.
> 
> @kabirsg I also fixed that issue with the trail showing up over the tree.

It looks great now thanks Owen!

File: src/components/Header.tsx
Comment (Krish120003) [inline src/components/Header.tsx:1]: If you've created  a component for this, could you apply using this to the rest of the header/titles as well please

**Review by Krish120003 — CHANGES_REQUESTED** [review]
@owengretzinger Please install and setup the appropriate font for this section, otherwise this looks great!

https://fontsource.org/fonts/gochi-hand

**Comment by owengretzinger** [comment]
Updated other headers and added the font (see updated screenshots)

File: package.json [hunk 1]
```diff
@@ -20,6 +20,7 @@
     "class-variance-authority": "^0.7.0",
     "clsx": "^2.1.1",
     "geist": "^1.3.1",
+    "local": "link:next/font/local",
```
Comment (Krish120003) [inline package.json]: Remove this dep

File: src/pages/index.tsx [hunk 2]
```diff
@@ -1,15 +1,31 @@
 import Sponsors from "~/sections/Sponsors";
 import FAQ from "~/sections/FAQ";
 import Footer from "~/sections/Footer";
+import JoinGrowingCommunity from "~/sections/JoinGrowingCommunity";
 import Hero from "~/sections/Hero";
+import localFont from "next/font/local";
+
+const gochi = localFont({
```
Comment (Krish120003) [inline src/pages/index.tsx]: https://fontsource.org/fonts/gochi-hand/install Switch to this method of installation to be consistent with how other fonts are intalled

File: src/components/Header.tsx [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+const Header: React.FC<HeaderProps> = ({ title, className, ...props }) => {
+  return (
+    <h2
+      className={clsx(
+        "text-display text-center text-4xl font-black text-white drop-shadow-[0_4px_4px_rgba(0,0,0,0.25)] lg:text-8xl",
+        className,
+      )}
+      {...props}
+    >
+      {title}
```
Comment (Krish120003) [inline src/components/Header.tsx]: Instead of taking title as a prop, why not pass in children as is? That way any word specific styling can be still applied without much changes

**Review by Krish120003 — CHANGES_REQUESTED** [review]

**Comment by owengretzinger** [comment]
fixed @Krish120003

**Review by Krish120003 — CHANGES_REQUESTED** [review]
image

There's this awkward break in colors on mobile (and on some desktop sizes as well). Could you adjust the background to be a gradient so this doesn't happen?

**Comment by owengretzinger** [comment]
@Krish120003 I changed the background color to make it match more. However, there's still a little line, which I think is because of a shadow baked into a hero section svg.

The Figma design also has a subtle gradient throughout the whole page. But I think that these things are outside the scope of this PR. Let me know what you think.

image

**Review by Krish120003 — APPROVED** [review]
